CONTRIBUTING.md

```diff
@@ -142,6 +144,12 @@ we would appreciate if your pull request applies to the following points:
       has been signed as well as commits have been signed-off and
     * _Continuous Integration_ performing various test conventions.
 
+### Add Documentation
+
+Every decision record, launcher, extension, or any type of module has to provide documentation that covers at least
+one markdown file with necessary information. Please find appropriate templates that should
+be used in [this directory](docs/templates).
```

Comment:
as the `CONTRIBUTING.md` is not within the `docs/templates` I wouldn't write `this`. I would make it more explictly:
```suggestion
be used in the [`docs/templates`  directory](docs/templates).
```